game/recastnavigation_bin.ksy

```diff
@@ -0,0 +1,283 @@
+meta:
+  id: recastnavigation_bin
+  file-extension: bin
+  license: CC0-1.0
+  endian: le
+doc: this spec can be used to parse recastnavigation binary files
+doc-ref: https://masagroup.github.io/recastdetour
+seq:
+  - id: header
+    type: navmesh_set_header
+  - id: tiles
+    type: navmesh_tile
+    repeat: expr
+    repeat-expr: header.num_tiles
+types:
+  navmesh_set_header:
```

Comment:
This type is only being used once, and when it's used, it actually makes accessing things harder than it should have been (i.e. you need to use `header.num_tiles` instead of just `num_tiles`). In general, this is [considered an antipattern](https://doc.kaitai.io/ksy_style_guide.html#_c_struct_as_header).

Please consider just embedding it into the top-level type?